uiowa/uiowa01
# Feature: Collegiate theme updates

This feature branch addresses issues #188, #163, #162, #156,  #135, #191 

# Testing

#188    

1. drush @theming.local si collegiate_basic
2. Add basic page content item with lots of body text
3. Visit the page to see if body content width is set to 74.75% of page width
4. You might need to refresh the page after saving the page
5. Also edit using layout builder and add a "Basic Text" block to a one-column layout with lots of text. 
6. Save the page and verify that the content width for that text block is set to 74.75%. 

#163

1. drush @theming.local si collegiate_basic
2. Add basic page content item
3. Visit the page and edit using layout builder
4. Add 1, 2, 3 column layout sections and add content
5. Verify that the width you have set in the theme settings is being used on each layout
6. Adjust layout section width settings to full or fixed container to override the theme settings

![Screen Shot 2019-11-07 at 3 18 20 PM](https://user-images.githubusercontent.com/1036433/68428454-d7b33680-0171-11ea-8b70-c9a5d3840ad6.png)

#162

1. drush @theming.local si collegiate_basic
2. Create a view
3. Verify that view page width reflects the width selected in theme settings
4. You might not see a page title because of #187 

#135 

1. drush @theming.local si collegiate_basic
2. Create a basic page
3. In a new browser, verify that the page title and breadcrumbs have spacing
4. Enable entity_types_news content type and create a news item
5. In a new browser, verify that there is a enough space above and below the content 
6. You might not see the breadcrumbs or page title because of issue #186

#156

1. drush @theming.local si collegiate_basic
2. Add basic page content item
3. Visit the page and edit using layout builder
4. Add 1, 2, 3 column layout sections and add custom blocks to each layout
5. Save the page and verify that there is an equal amount of spacing between each column

#191 

1. Verify that the default installation without a site slogan has reduced spacing between the wordmark and the site name. 
2. Add a site slogan to test whether extra spacing is added back. 
3. Change the background color to gold in the theme settings to test whether the spacing between the wordmark and the site name expands. 

![Screen Shot 2019-11-08 at 2 56 29 PM](https://user-images.githubusercontent.com/1036433/68509965-f419a680-0237-11ea-845e-c461fbda82b4.png)

![Screen Shot 2019-11-08 at 2 56 55 PM](https://user-images.githubusercontent.com/1036433/68509997-04318600-0238-11ea-904a-c33e8239a0c2.png)

#190 

1. Verify with AInspector WCAG toolbar that theme does not have any violations.

![Screen Shot 2019-11-11 at 1 56 28 PM](https://user-images.githubusercontent.com/1036433/68616660-19522300-048b-11ea-9bf7-7b1a3664d284.png)
